<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CArmImage</class>
 <widget class="QWidget" name="CArmImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>623</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>328</width>
    <height>623</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>328</width>
     <height>420</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>328</width>
     <height>420</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>328</width>
     <height>420</height>
    </size>
   </property>
   <widget class="QLabel" name="brightnessText">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>155</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>36</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>36</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="brightnessLabel">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>125</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="CArmLabel" name="sharpen">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>219</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="CArmLabel" name="denoise">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>297</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="contrastLabel">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>43</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="contrastValue">
    <property name="geometry">
     <rect>
      <x>273</x>
      <y>28</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>50%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="contrastText">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>72</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>36</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>36</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="TouchPanelSlider" name="contrastSlider">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>47</y>
      <width>216</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>216</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>216</width>
      <height>22</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal{
height: 4px;
background-color:rgba(66,80,96,1) ;
border: 1px solid #566576;
border-radius: 0px;
padding-left:-1px;
padding right:-1px;}
        
QSlider::sub-page:horizontal{
height: 4px;
background: transparent;
border: 1px solid transparent;
border-radius: 2px;
}
  
QSlider::handle:horizontal
{
    width: 22px;
    background-color:rgba(255,255,255,1);
    margin-top: -10px;
    margin-bottom: -10px;
border-radius: 5px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="TouchPanelSlider" name="brightnessSlider">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>130</y>
      <width>216</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>216</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>216</width>
      <height>22</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal{
height: 4px;
background-color:rgba(66,80,96,1) ;
border: 1px solid #566576;
border-radius: 0px;
padding-left:-1px;
padding right:-1px;}
        
QSlider::sub-page:horizontal{
height: 4px;
background: transparent;
border: 1px solid transparent;
border-radius: 2px;
}
  
QSlider::handle:horizontal
{
    width: 22px;
    background-color:rgba(255,255,255,1);
    margin-top: -10px;
    margin-bottom: -10px;
border-radius: 5px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="brightnessValue">
    <property name="geometry">
     <rect>
      <x>273</x>
      <y>111</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>50%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="sharpenSlider">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>219</y>
      <width>216</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>216</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>216</width>
      <height>22</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal{
height: 4px;
background-color:rgba(66,80,96,1) ;
border: 1px solid #566576;
border-radius: 0px;
padding-left:-1px;
padding right:-1px;}
        
QSlider::sub-page:horizontal{
height: 4px;
background: transparent;
border: 1px solid transparent;
border-radius: 2px;
}
  
QSlider::handle:horizontal
{
    width: 22px;
    background-color:rgba(255,255,255,1);
    margin-top: -10px;
    margin-bottom: -10px;
border-radius: 5px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="sharpenValue">
    <property name="geometry">
     <rect>
      <x>273</x>
      <y>200</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>50%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="denoiseValue">
    <property name="geometry">
     <rect>
      <x>273</x>
      <y>280</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>50%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="denoiseSlider">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>299</y>
      <width>216</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>216</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>216</width>
      <height>22</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal{
height: 4px;
background-color:rgba(66,80,96,1) ;
border: 1px solid #566576;
border-radius: 0px;
padding-left:-1px;
padding right:-1px;}
        
QSlider::sub-page:horizontal{
height: 4px;
background: transparent;
border: 1px solid transparent;
border-radius: 2px;
}
  
QSlider::handle:horizontal
{
    width: 22px;
    background-color:rgba(255,255,255,1);
    margin-top: -10px;
    margin-bottom: -10px;
border-radius: 5px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="metaLabel">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>360</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>350</y>
      <width>216</width>
      <height>38</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>216</width>
      <height>38</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>216</width>
      <height>38</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="CArmLabel" name="zero">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>54</width>
         <height>38</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="CArmLabel" name="one">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>54</width>
         <height>38</height>
        </size>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="CArmLabel" name="two">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>54</width>
         <height>38</height>
        </size>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="CArmLabel" name="three">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>54</width>
         <height>38</height>
        </size>
       </property>
       <property name="text">
        <string>3</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="denoiseGear">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>297</y>
      <width>8</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="CArmLabel" name="denoiseSelectGear">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>314</y>
      <width>7</width>
      <height>7</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="CArmLabel" name="sharpenSelectGear">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>236</y>
      <width>7</width>
      <height>7</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="sharpenGear">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>219</y>
      <width>8</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QLabel" name="metalText">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>390</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>36</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>36</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_7" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>328</width>
     <height>202</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>328</width>
     <height>202</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>328</width>
     <height>202</height>
    </size>
   </property>
   <widget class="CArmLabel" name="metaRevise">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>13</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>52</width>
      <height>52</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>52</width>
      <height>52</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="reviseText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>72</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>智能金属校正</string>
    </property>
   </widget>
   <widget class="CArmLabel" name="negative">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>13</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>52</width>
      <height>52</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>52</width>
      <height>52</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="CArmLabel" name="rotate">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>13</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>52</width>
      <height>52</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>52</width>
      <height>52</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="negativeText">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>72</y>
      <width>52</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>52</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>52</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>负片</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotateText">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>72</y>
      <width>52</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>52</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>52</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>旋转</string>
    </property>
   </widget>
   <widget class="QLabel" name="levelText">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>169</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>水平翻转</string>
    </property>
   </widget>
   <widget class="CArmLabel" name="level">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>52</width>
      <height>52</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>52</width>
      <height>52</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="verticalText">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>169</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>垂直翻转</string>
    </property>
   </widget>
   <widget class="CArmLabel" name="vertical">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>110</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>52</width>
      <height>52</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>52</width>
      <height>52</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="resetText">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>169</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>重置图像</string>
    </property>
   </widget>
   <widget class="CArmLabel" name="reset">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>52</width>
      <height>52</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>52</width>
      <height>52</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="CArmLabel" name="ValueLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>24</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>328</width>
     <height>1</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>328</width>
     <height>1</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>328</width>
     <height>1</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CArmLabel</class>
   <extends>QLabel</extends>
   <header location="global">CArmControl.h</header>
  </customwidget>
  <customwidget>
   <class>TouchPanelSlider</class>
   <extends>QSlider</extends>
   <header>CArmTouchPanelSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
